flowmachine/flowmachine/core/cache.py

```diff
@@ -363,31 +365,45 @@ def get_query_object_by_id(connection: "Connection", query_id: str) -> "Query":
 
 
 def get_cached_query_objects_ordered_by_score(
-    connection: "Connection"
+    connection: "Connection", protected_period: Optional[int] = None
 ) -> List[Tuple["Query", int]]:
     """
     Get all cached query objects in ascending cache score order.
 
     Parameters
     ----------
     connection : Connection
+    protected_period : int, default None
+        Optionally specify a number of seconds within which cache entries are excluded. If None,
+        the value stored in cache.cache_config will be used. Set to -1 to ignore cache protection
```

Comment:
I'd prefer to say 'set to a value less than 0', otherwise this gives the incorrect impression that -1 is a special value.